src/neo4j_graphrag/llm/ollama_llm.py

```diff
@@ -12,19 +12,25 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-from typing import Any, Optional
+from typing import Any, Optional, Sequence, TYPE_CHECKING, cast
+
+from pydantic import ValidationError
 
 from neo4j_graphrag.exceptions import LLMGenerationError
 
 from .base import LLMInterface
-from .types import LLMResponse
+from .types import LLMResponse, SystemMessage, UserMessage, MessageList
+
+if TYPE_CHECKING:
+    from ollama import Message
```

Comment:
Why isn't this working? It's giving `NameError: name 'Message' is not defined`.

Reply:
[17db6b1](https://github.com/neo4j/neo4j-graphrag-python/pull/234/commits/17db6b18de6708ffac48fb557d5991b387d2fac2)